packages/browser/src/eventbuilder.ts

```diff
@@ -278,6 +278,9 @@ export function eventFromString(
       event.exception = {
         values: [{ value: input, stacktrace: { frames } }],
       };
+      event.threads = {
+        values: [{ stacktrace: { frames } }],
```

Comment:
I don't understand how this is fixing the problem here - doesn't this just duplicate the information?

Reply:
I agree, the [PR](https://github.com/getsentry/sentry-javascript/issues/5879#issue-1396447333) has a description of what needs to be fixed.

Reply:
So, previously when debugging this, when calling `captureMessage`, the `syntheticException` was null, this is written in the issue description.
Apparently something changed or I was using an older version of the JS SDK, now the `syntheticException` isn't null anymore even for `captureMessage` and message events contain an exception with a stack trace, IMO this is wrong.
`captureMessage` should contain a thread with the stack trace and not an exception with a stack trace.

Reply:
@AbhiPrasad can you double-check if that makes sense now? thanks.